### PR TITLE
WIP: collected during relation

### PR DESCRIPTION
**Pull request by ddooley**
A relation between a material entity and an occurent in which the specimen is collected during the existence of the occurent.

### CODE REVIEW

**Comment by wdduncan** [comment]
'occurent' should be 'occurrent' (needs another 'r').

Perhaps be consistent in the use of 'material entity'? Information about 'specimen' could be in a one sentence gloss. E.g.:  
```
A relation between a material entity and an occurrent in which the material entity is collected during the existence of the occurrent. The material entity is a specimen that collected for scientific purposes.
```

**Comment by ddooley** [comment]
I made suggested change.  This can now be approved and closed!

**Review by anitacaron — DISMISSED** [review]
Technically approved. It still needs ontology approval.

**Review by anitacaron — DISMISSED** [review]
Technically approved. It still needs ontology approval.

File: src/ontology/ro-edit.owl [hunk 1]
```diff
@@ -6612,6 +6613,17 @@ AnnotationAssertion(oboInOwl:creation_date obo:RO_0017009 "2023-03-29T17:06:06Z"
...
+# Object Property: obo:RO_0017502 (collected during)
+
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017502 "A thorax specimen was collected during the larval stage."@en)
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017502 "An oyster was collected during low tide."@en)
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0017502 "A relation between a material entity and an occurrent in which the material entity is collected during the existence of the occurrent. The material entity is a specimen that is collected for scientific purposes."@en)
+AnnotationAssertion(terms:contributor obo:RO_0017502 <https://orcid.org/0000-0002-8844-9165>)
+AnnotationAssertion(terms:date obo:RO_0017502 "2023-09-13T04:12:32Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:RO_0017502 "collected during"@en)
+ObjectPropertyDomain(obo:RO_0017502 obo:BFO_0000040)
+ObjectPropertyRange(obo:RO_0017502 obo:BFO_0000003)
```
Comment (cmungall) [inline src/ontology/ro-edit.owl:6625]: general observation: this is occurrent, which is above the shoreline in COB. See #780. This comment is not to block the PR: clearly we have a precedent for using occurrent already

File: src/ontology/ro-edit.owl [hunk 2]
```diff
@@ -6612,6 +6613,17 @@ AnnotationAssertion(oboInOwl:creation_date obo:RO_0017009 "2023-03-29T17:06:06Z"
 AnnotationAssertion(oboInOwl:inSubset obo:RO_0017009 obo:RO_0002259)
 AnnotationAssertion(rdfs:label obo:RO_0017009 "is owned by"@en)
 
+# Object Property: obo:RO_0017502 (collected during)
+
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017502 "A thorax specimen was collected during the larval stage."@en)
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017502 "An oyster was collected during low tide."@en)
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0017502 "A relation between a material entity and an occurrent in which the material entity is collected during the existence of the occurrent. The material entity is a specimen that is collected for scientific purposes."@en)
```
[the remark this answers is not in the document]
Comment (ddooley) [reply src/ontology/ro-edit.owl:6653]: Done!

**Review by ddooley — COMMENTED** [review]
Good to go!

File: src/ontology/ro-edit.owl [hunk 3]
```diff
@@ -44,6 +44,7 @@ Declaration(Class(obo:BFO_0000031))
 Declaration(Class(obo:CHEBI_50906))
 Declaration(Class(obo:COB_0001000))
 Declaration(Class(obo:OGMS_0000031))
+Declaration(Class(obo:PATO_0001470))
```
Comment (anitacaron) [inline src/ontology/ro-edit.owl:47]: Why this PATO term was included in the edit file?

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/ro-edit.owl (hunk 3 above)
Comment (anitacaron) [reply src/ontology/ro-edit.owl:47]: I don't think so. :) I've removed it anyway.

**Review by anitacaron — APPROVED** [review]
Technically, I approve. Still need to have an ontology review,

**Review by cmungall — APPROVED** [review]

**Pull request merged**